### ISSUE TITLE
Decrease/increase font size in Gesture manager: tap/hold corner & pdf

### ISSUE THREAD

**Issue by gerhaher** [koreader/koreader#5341]
* KOReader version: v2019.08-101-ga3acc66_2019-09-07/  
* Device: KA1

#### Issue
Regarding font size in Gesture manager:

1. Tap/hold corner not working
2. Pdf:s (in reflow mode) no gestures are working.

**Comment by NiLuJe** [comment]
The corner thing might just be a side effect of the touch screen sensitivity being basically for shit on the edges of a KA1...

**Comment by NiLuJe** [comment]
(I certainly have no issues with at least corner taps on both a H2O and a Forma).

**Comment by Frenzie** [comment]
@NiLuJe Actually I think the font size increase/decrease action doesn't seem to do much, regardless of corner sensitivity.

**Comment by NiLuJe** [comment]
Yeah, that's also quite likely, I was wondering about those yesterday, but
didn't check ;).

**Comment by Frenzie** [comment]
It does work based on the edge swipes. It's probably just the tap value that's too small to make an impact or something.

**Comment by Frenzie** [comment]
There are different events, that explains it.

`AdjustFontSize` for crengine
https://github.com/koreader/koreader/blob/b23af97914326fde91c27c2bb0cef46b9fa72caa/frontend/apps/reader/modules/readerfont.lua#L323-L349

And `FontSizeUpdate` for kopt.
https://github.com/koreader/koreader/blob/7b1ae4e486b9a920c0553cd654adc08668265116/frontend/apps/reader/modules/readerview.lua#L813-L815

I think `0.05` in kopt is about the same as `1` in crengine.
